src/main/java/gregtech/api/recipes/RecipeMap.java

```diff
@@ -328,8 +336,9 @@ protected ValidationResult<Recipe> postValidateRecipe(@Nonnull ValidationResult<
             }
             recipeStatus = EnumValidationResult.INVALID;
         }
-        boolean emptyOutputs = recipe.getOutputs().isEmpty() && recipe.getFluidOutputs().isEmpty() && recipe.getChancedOutputs().isEmpty();
-        if (recipe.getEUt() > 0 && emptyOutputs) {
+        boolean emptyOutputs = !this.allowEmptyOutput && recipe.getEUt() > 0 &&
```

Comment:
Should the negative EUt check be removed now in order to require enabling empty outputs?